src/Updater.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Tuf\ComposerIntegration;
+
+use Psr\Http\Message\StreamInterface;
+use Tuf\Client\Updater as BaseUpdater;
+use Tuf\Metadata\TargetsMetadata;
+
+/**
+ * Defines an updater that exposes additional information about TUF targets.
+ *
+ * @internal
+ */
+class Updater extends BaseUpdater
```

Comment:
I still think we should give this a name that means something, like `ComposerCompatibleUpdater` or something.

Reply:
Done!